test/functional/page_objects/time_picker.js

```diff
@@ -188,11 +188,10 @@ export function TimePickerPageProvider({ getService, getPageObjects }) {
     }
 
     async getTimeDurationInHours() {
-      const DEFAULT_DATE_FORMAT = 'MMM D, YYYY @ HH:mm:ss.SSS';
       const { start, end } = await this.getTimeConfigAsAbsoluteTimes();
 
-      const startMoment = moment(start, DEFAULT_DATE_FORMAT);
-      const endMoment = moment(end, DEFAULT_DATE_FORMAT);
+      const startMoment = moment(Date.parse(start));
```

Comment:
I don't think we want to use `Date.parse`, even MDN developer guides recommend against it - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse. 

Lets, lets just update the format string to the correct value - [YYYY-MM-DD HH:mm:ss.SSS](https://github.com/elastic/eui/blob/master/src/components/date_picker/super_date_picker/date_popover/absolute_tab.js#L11)

Reply:
Ah, I tried it with `yyyy-mm-dd HH:mm:ss.SSS` and it was ignoring date (returning -20.8), so I decided to go with `Date.parse`. Thank you, I will change it.